_includes/releases/v22.1/v22.1.12.md

```diff
@@ -0,0 +1,104 @@
+- Fixed a bug which, in rare cases, could result in a changefeed missing rows which occur around the time of a split in writing transactions which take longer than the closed timestamp target duration (defaults to 3s). [#91749][#91749]
+- Add leading zeros to fingerprint IDs with less than 16 characters. [#91959][#91959]
+- Fixed a bug introduced in 20.2 that could cause filters to be dropped from a query plan with many joins in rare cases. [#91654][#91654]
+- Fixed an unhandled error that could happen if `ALTER DEFAULT PRIVILEGES` was run on the system database. [#92083][#92083]
+- Reduced the amount that RESTORE over-splits. [#91141][#91141]
+- Fixed a bug causing changefeeds to fail when a value is deleted while running on a non-primary column family with multiple columns. [#91953][#91953]
+- Strip quotation marks from database and table names to correctly query for index usage statistics. [#92282][#92282]
+- Fixed the statement activity page so that it no longer shows multi-statement implicit transactions as "explicit." [#92430][#92430]
+- Fixed a bug existing since 20.2 that could cause incorrect results in rare cases for queries with inner joins and left joins. For the bug to occur, the left join had to be in the input of the inner join and the inner join filters had to reference both inputs of the left join, and not filter NULL values from the right input of the left join. Additionally, the right input of the left join had to contain at least one join, with one input not referenced by the left join's ON condition. [#92103][#92103]
+- When configured to true, the `sql.metrics.statement_details.dump_to_logs` cluster setting no longer causes a mutex deadlock. [#92278][#92278]
+- Fixdc incorrect cancellation logic when attempting to detect stuck range feeds. [#92702][#92702]
+- Fixed an internal error when comparing a tuple type with a non-tuple type. [#92714][#92714]
+- `attidentity` for `GENERATED BY DEFAULT AS IDENTITY` column should be `d`.  <what was there before: Previously, ...> <why it needed to change: This was inadequate because ...> <what you did about it: To address this, this patch ...> [#92835][#92835]
+- CockroachDB previously could incorrectly evaluate queries that performed left semi and left anti "virtual lookup" joins on tables in `pg_catalog` or `information_schema`. These join types can be planned when a subquery is used inside of a filter condition. The bug was introduced in 22.1.0 and is now fixed. [#92881][#92881]
+
+<h3 id="v22-1-12-performance-improvements">Performance improvements</h3>
+
+- GC score triggering mvcc GC run is lowered. GC Score is ratio is computed from mvcc stats and uses ratio of live objects and estimated garbage age to estimate collectability of old data. Reduced score will trigger GC earlier lowering interval between runs 3 times, giving 2 times reduced peak garbage usage at the expense of 30% increase of wasteful data scanning on constantly updated data. [#92816][#92816]
```

Comment:
I think this needs some more heavy edits, because it's hard to understand. Maybe: 

To protect against unexpected situations where garbage collection would trigger too frequently, the GC score cooldown period has been lowered. The GC score ratio is computed from MVCC stats and uses ratio of live objects and estimated garbage age to estimate collectability of old data. The reduced score will trigger GC earlier, lowering interval between runs 3 times, giving 2 times reduced peak garbage usage at the expense of 30% increase of wasteful data scanning on constantly updated data.